wazo_acceptance/steps/stats.py

```diff
@@ -37,3 +37,40 @@ def then_contact_center_stats_for_queue_1(context, queue_name):
         actual_value = total_stats[stat_name]
         assert actual_value == expected_value, \
             f'expected {stat_name} = {expected_value}, got {actual_value}'
+
+
+@then(u'contact center stats for agent "{agent_number}" in the current hour are')
+def then_contact_center_stats_for_agent(context, agent_number):
+    now = datetime.now()
+    last_hour = datetime(now.year, now.month, now.day, now.hour, 0, 0)
+
+    agent_id = context.helpers.agent.get_by(number=agent_number)['id']
+    agent_stats = context.call_logd_client.agent_statistics.get_by_id(
+        agent_id=agent_id,
+        from_=last_hour.isoformat(),
```

Comment:
See the queue tests. But you should specify timezone:
* when jenkins run test, the UTC will be in the next day and you will have no stats
* avoid false positive on the day we change hour